### PR TITLE
:construction: Creat form.py and class FormForms and update form_view() to handle POST request

### PR DESCRIPTION
**Pull request by Serg-Bartosh**
If all data is correct 200, if data is incorrect or repeated 400*

### CODE REVIEW

File: forms/views.py [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
+    if request.method == "POST":
+        data = FormForms(request.POST)
+
+        if data.is_valid():
+            Form.objects.create(**data.cleaned_data)
+            return HttpResponse()
 
-    forms_query_set = Form.objects.all()
-    forms_json = list(forms_query_set.values())
+        return HttpResponse(status=500)
```
Comment (s373r) [inline forms/views.py:21]: Use `400` (Bad Request) for that -- 500 is for **unexpected** cases. Invalid input data from users is an **expected** case

File: forms/form.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from django import forms
+
+
+class FormForms(forms.Form):
```
Comment (s373r) [inline forms/form.py:4]: 1) Please this file at `forms/dtos.py`
2) `FormForms` -> `FormCreateDto`*

Please google about DTOs or ask me in Telegram

**Review by s373r — CHANGES_REQUESTED** [review]

File: forms/form.py (hunk 2 above)
Comment (Serg-Bartosh) [reply forms/form.py]: Did it with love

File: forms/views.py (hunk 1 above)
Comment (Serg-Bartosh) [reply forms/views.py]: Done

File: forms/form.py (hunk 2 above)
Comment (s373r) [reply forms/form.py]: > Please this file at forms/dtos.py

do not finished

Comment (Serg-Bartosh) [reply forms/form.py]: This renaming is cooler than cool eggs

**Review by s373r — APPROVED** [review]
Hell yeah!

**Pull request merged**